docs/infrastructure/workers/index.md

```diff
@@ -47,4 +47,6 @@ Default pools attached to cloud targets allow co-location of workers and targets
 
 ## Multiple Projects Run Simultaneously on Workers
 
-The built-in worker can run many steps simultaneously.  For example, the task cap determines how many deployments can run simultaneously and those deployments could run many steps concurrently on the built-in worker.  External workers are the same.  External workers will run steps from different projects simultaneously.  This means external workers keep the behavior of the built-in worker, including that a step doesn't have exclusive access to the worker, which could allow one project to access the working folder of another project.
+The built-in worker can run many steps simultaneously.  For example, the [task cap](/docs/support/increase-the-octopus-server-task-cap.md) determines how many deployments can run simultaneously and those deployments could run many steps concurrently on the built-in worker.  External workers are the same.  External workers will run steps from different projects simultaneously.  This means external workers keep the behavior of the built-in worker, including that a step doesn't have exclusive access to the worker, which could allow one project to access the working folder of another project.
+
+The number of parallel worker invocations any one deployment task will make is controlled by the [Octopus System Variable](/docs/deployment-process/variables/system-variables.md) `Octopus.Action.MaxParallelism`.  For example, if your server task cap is 2 and the `Octopus.Action.MaxParallelism` variable is at its default value of 10, then you could be running 20 instances of the built-in worker concurrently (it's actually more because there could also be parallel steps within those deployment tasks).  If external workers are added to the default pool, then the workload is shared across those workers: a single worker will get the whole workload that the built-in worker used to do, two workers might get half each, etc.  Remember that in an HA setup each node has a task cap and can invoke a built-in worker locally, so you'll need to provision workers to at least match your server nodes, otherwise, you'll be asking each worker to do the sum of what the HA nodes were previously doing.
```

Comment:
It might be worth breaking out the HA info into a subsection with a heading so it stands out for HA users.